usaspending_api/spending_explorer/tests/integration/test_spending_explorer.py

```diff
@@ -519,3 +519,25 @@ def test_agency_failure(client):
         data=json.dumps({"type": "agency", "filters": {"fy": "23", "quarter": "3"}}),
     )
     assert resp.status_code == status.HTTP_400_BAD_REQUEST
+
+
+@pytest.mark.django_db
+def test_object_budget_match(client):
+
+    models = copy.deepcopy(GLOBAL_MOCK_DICT)
+    for entry in models:
+        mommy.make(entry.pop("model"), **entry)
+
+    json_request = {"type": "budget_function", "filters": {"fy": "1600", "quarter": "1"}}
+
+    response = client.post(path=ENDPOINT_URL, content_type=CONTENT_TYPE, data=json.dumps(json_request))
+
+    assert response.status_code == status.HTTP_200_OK
+
+    json_response_1 = response.json()
+
+    json_request = {"type": "object_class", "filters": {"fy": "1600", "quarter": "1"}}
+
+    response = client.post(path=ENDPOINT_URL, content_type=CONTENT_TYPE, data=json.dumps(json_request))
+    json_response_2 = response.json()
+    assert json_response_1["results"][0]["amount"] == json_response_2["results"][0]["amount"]
```

Comment:
Have you ensured that this test would have failed without the change? Based on the GLOBAL_MOCK_DICT, I'm not sure our test data is rich enough to actually reproduce the error.